src/test/java/com/bbtutorials/users/dao/WorkoutDaoTests.java

```diff
@@ -0,0 +1,140 @@
+                .withAttributeDefinitions(
+                        new AttributeDefinition().withAttributeName("Id").withAttributeType(ScalarAttributeType.S))
+                .withProvisionedThroughput(new ProvisionedThroughput(5L, 5L))
+        );
+
+        workoutDao = new WorkoutDao(mapper);
+    }
+
+    public Workout createWorkout() {
+        Workout testWorkout = new Workout();
+
+        testWorkout.setUserEmail(testEmail);
+        testWorkout.setName(testName);
+        
+        Set<String> testExer = new HashSet<String>();
+        Set<Integer> testDays = new HashSet<Integer>();
+        
+        testExer.add("testexercise1");
+        testExer.add("testexercise2");
+        testWorkout.setExercises(testExer);
+        
+        testDays.add(1);
+        testDays.add(2);
+        testWorkout.setDaysOfWeek(testDays);
+
+        return testWorkout;
+    }
+
+
+    @Test
+    public void testPutGetRemoveWorkout() {
+        workoutDao.putWorkout(createWorkout());
+
+        List<Workout> workouts = workoutDao.getWorkouts(testEmail);
+        
+        Set<Integer> checkDays = workouts.get(0).getDaysOfWeek();
+        Set<String> checkExer = workouts.get(0).getExercises();
+        assertEquals(1, workouts.size());
+        assertEquals(testName, workouts.get(0).getName());
+
+        assertEquals(testEmail, workouts.get(0).getUserEmail());
+        assertEquals(2, checkDays.size());
+        assertEquals(2, checkExer.size());
+        assertEquals(true, checkDays.contains(2));
+        assertEquals(true, checkDays.contains(1));
+        assertEquals(true, checkExer.contains("testexercise1"));
+        assertEquals(true, checkExer.contains("testexercise2"));
+        
+        workoutDao.removeWorkout(workouts.get(0));
+
+        workouts = workoutDao.getWorkouts(testEmail);
+
+        assertTrue(workouts.isEmpty());
+    }
+
+    @Test
+    public void testMultipleWorkouts() {
+        workoutDao.putWorkout(createWorkout());
+        Workout second = createWorkout();
+
+        workoutDao.putWorkout(second);
+
+        List<Workout> workouts = workoutDao.getWorkouts(testEmail);
+        
+
+        assertEquals(2, workouts.size());
+        
+        for (Workout toRemove : workouts) {
+            workoutDao.removeWorkout(toRemove);
+        }
+    }
+
+    @Test
+    public void testUpdateWorkout() {
+        workoutDao.putWorkout(createWorkout());
+
+        List<Workout> workouts = workoutDao.getWorkouts(testEmail);
+        Set<Integer> testDays2 = new HashSet<Integer>();
+        
+        testDays2.add(3);
+        testDays2.add(4);
+        
+        Workout updatedWorkout = workouts.get(0);
+        updatedWorkout.setDaysOfWeek(testDays2);
+
+        workoutDao.updateWorkout(updatedWorkout);
+
+        workouts = workoutDao.getWorkouts(testEmail);
+
+        Set<Integer> checkDays = workouts.get(0).getDaysOfWeek();
+        assertEquals(true, checkDays.contains(3));
```

Comment:
This is what I meant: `assertEquals(testDays2, workouts.get(0).getDaysOfWeek());`

Reply:
Same with above checks